feature-detects/mediastream.js

```diff
@@ -0,0 +1,13 @@
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', function(){
+        var isMediaRecorderSupported = false;
+
+        try {
+            MediaRecorder;
+            isMediaRecorderSupported = true;
+        } catch (err) {
+            console.log("no MediaRecorder");
+        }
+        console.log(isMediaRecorderSupported);
```

Comment:
you don't want to use console.log in the actual commit. Its very useful while you are making the detect, but it isn't useful for those consuming your work